cmd/faucet-v2/metrics.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	batchSendCount = promauto.NewCounter(prometheus.CounterOpts{
+		Namespace: "faucet",
+		Name:      "batch_send_total",
+		Help:      "The total number of sent batches (success or error)",
+	})
+
+	batchSize = promauto.NewGauge(prometheus.GaugeOpts{
+		Namespace: "faucet",
+		Name:      "batch_size",
+		Help:      "The size of the batch of addresses waiting for tokens",
+	})
+
+	reqCount = promauto.NewCounter(prometheus.CounterOpts{
+		Namespace: "faucet",
+		Name:      "req_total",
+		Help:      "The total number of faucet requests",
+	})
+
+	reqInvalidIpCount = promauto.NewCounter(prometheus.CounterOpts{
+		Namespace: "faucet",
+		Name:      "req_invalid_ip_total",
+		Help:      "The total number of failed requests for invalid IP",
+	})
+
+	reqInvalidAddrCount = promauto.NewCounter(prometheus.CounterOpts{
+		Namespace: "faucet",
+		Name:      "req_invalid_addr_total",
+		Help:      "The total number of failed requests for invalid address",
+	})
+
+	reqInvalidCaptchaCount = promauto.NewCounter(prometheus.CounterOpts{
+		Namespace: "faucet",
+		Name:      "req_invalid_captcha_total",
+		Help:      "The total number of failed requests for invalid captcha",
+	})
+
+	reqErrorCount = promauto.NewCounter(prometheus.CounterOpts{
+		Namespace: "faucet",
+		Name:      "req_error_count",
```

Comment:
I think usually these are in Prometheus with `_total` suffix

Reply:
done!